apps/client/src/app/components/holding-detail-dialog/holding-detail-dialog.component.ts

```diff
@@ -557,6 +558,32 @@ export class GfHoldingDetailDialogComponent implements OnDestroy, OnInit {
     this.dialogRef.close();
   }
 
+  public onCloseHolding() {
+    const today = new Date();
+
+    const activity: CreateOrderDto = {
+      accountId: null,
```

Comment:
We could apply the the account id as well, but only if `this.accounts.length` is `1`. What do you think?

Reply:
That makes sense. If it's the only account for the asset then it should be applied automatically.